lxd/networks.go

```diff
@@ -558,11 +574,80 @@ func (n *network) Start() error {
 
 	// Configure IPv4
 	if !shared.StringInSlice(n.config["ipv4.address"], []string{"", "none"}) {
+		// Parse the subnet
```

Comment:
These two blocks of ipv4 and ipv6 are almost identical. Is there some nice way to factor them out and re-use stuff?